jmespath-core/src/main/java/io/burt/jmespath/BaseRuntime.java

```diff
@@ -29,25 +32,23 @@
     '\"', '\"'
   );
 
+  private final RuntimeConfiguration configuration;
   private final FunctionRegistry functionRegistry;
   private final NodeFactory<T> nodeFactory;
 
   /**
    * Create a new runtime with a default function registry.
    */
   public BaseRuntime() {
-    this(null);
+    this(RuntimeConfiguration.defaultConfiguration());
   }
 
   /**
-   * Create a new runtime with a custom function registry.
+   * Create a new runtime with configuration.
```

Comment:
There could be a `@see RuntimeConfiguration` tag in the doc block though, but you get the link to `RuntimeConfiguration` on the method signature anyway.